autoreject/tests/test_autoreject.py

```diff
@@ -71,45 +76,91 @@ def test_autoreject():
     assert_raises(ValueError, ar.fit, X)
     ar = GlobalAutoReject(n_times=n_times)
     assert_raises(ValueError, ar.fit, X)
-    ar = GlobalAutoReject(n_channels=n_channels, n_times=n_times,
-                          thresh=40e-6)
-    ar.fit(X)
+    ar_global = GlobalAutoReject(
+        n_channels=n_channels, n_times=n_times, thresh=40e-6)
+    ar_global.fit(X)
 
     reject = get_rejection_threshold(epochs)
     assert_true(reject, isinstance(reject, dict))
 
     param_name = 'thresh'
     param_range = np.linspace(40e-6, 200e-6, 10)
-    assert_raises(ValueError, validation_curve, ar, X, None,
+    assert_raises(ValueError, validation_curve, ar_global, X, None,
                   param_name, param_range)
 
     ##########################################################################
     # picking AutoReject
-    picks_list = [('mag', False, []), (False, True, include)]
+    picks_list = [('mag', True, []),
+                  ]
     for ii, (meg, eeg, include_) in enumerate(picks_list):
 
         picks = mne.pick_types(
             epochs.info, meg=meg, eeg=eeg, stim=False, eog=False,
             include=include_, exclude=[])
+        ch_types = []
+        if meg == 'mag' or meg == 'meg':
+            ch_types.append('mag')
+        if meg == 'grad' or meg == 'meg':
+            ch_types.append('grad')
+        if eeg is True:
+            ch_types.append('eeg')
 
         ar = LocalAutoReject(picks=picks)
         assert_raises(NotImplementedError, validation_curve, ar, epochs, None,
                       param_name, param_range)
 
-        ar = LocalAutoRejectCV(cv=3, picks=picks)
+        ar = LocalAutoRejectCV(cv=3, picks=picks,
+                               n_interpolates=[1, 2],
+                               consensus_percs=[0.5, 1])
         assert_raises(AttributeError, ar.fit, X)
         assert_raises(ValueError, ar.transform, X)
         assert_raises(ValueError, ar.transform, epochs)
 
-        ar.fit(epochs)
+        ar.fit(epochs_fit)
+        fix_log = ar.fix_log
+        bad_epochs_idx = ar.local_reject_.bad_epochs_idx_
+        good_epochs_idx = ar.local_reject_.good_epochs_idx_
```

Comment:
hum ... why is there `good_epochs_idx_` when we haven't transformed yet? This is counterintuitive, no?